archivebox/index/__init__.py

```diff
@@ -261,38 +246,30 @@ def write_main_index(links: List[Link], out_dir: str=OUTPUT_DIR, finished: bool=
 
     log_indexing_process_finished()
 
+@enforce_types
+def write_static_index(links: List[Link], out_dir: str=OUTPUT_DIR) -> None:
+    with timed_index_update(os.path.join(out_dir, JSON_INDEX_FILENAME)):
+        write_json_main_index(links)
+    with timed_index_update(os.path.join(out_dir, HTML_INDEX_FILENAME)):
+        write_html_main_index(links, out_dir=out_dir, finished=True)
+
+@enforce_types
+def get_empty_snapshot_queryset(out_dir: str=OUTPUT_DIR):
```

Comment:
Why do we need a function for this? Isn't `Snapshot.objects.none()` clearer on its own?

Reply:
`RuntimeError: Model class archivebox.core.models.Snapshot doesn't declare an explicit app_label and isn't in an application in INSTALLED_APPS.`. I need to re-run setup_django with check_db=True (which is what I do in that function).
Once django is initialized before anything else, we could change it with no issues.